apps/server/src/routes/files/index.ts

```
@@ -1,208 +1,349 @@
+
+		const drive = await getDriveManagerForUser(user, c.req.raw.headers);
+		const res = await drive.listFiles(data.parentId, data.pageSize, data.returnedValues, data.pageToken);
+
+		if (!res.files) {
+			return c.json<ApiResponse>({ success: false, message: "Files not found" }, 404);
+		}
+
+		// Add tags to files
+		const filesWithTags = await Promise.all(
+			res.files.map(async file => {
+				const tags = await tagService.getFileTags(file.id!, user.id);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)